src/MSBuild/XMake.cs

```diff
@@ -627,7 +627,7 @@ string commandLine
 
                     // Honor the low priority flag, we place our selves below normal
                     // priority and let sub processes inherit that priority.
-                    ProcessPriorityClass priority = lowPriority ? ProcessPriorityClass.BelowNormal : ProcessPriorityClass.Normal;
+                    ProcessPriorityClass priority = lowPriority ? ProcessPriorityClass.BelowNormal : Process.GetCurrentProcess().PriorityClass;
```

Comment:
Should we fetch the process object once? (we do it again immediately below?) I imagine this is another TLS lookup for no reason.

Reply:
It's also odd that we even bother to set the process priority at all if we are only going to set it to it's current value.

Reply:
Principal reason is that I'm a big fan of ternary operators, but I agree it doesn't make sense here. 😉